src/agents/der_aggregator.jl

```diff
@@ -20,6 +20,8 @@ mutable struct DERAggregator <: AbstractDERAggregator
     aggregation_friction::ParamArray
     # incentive level by year
     incentive_level::ParamArray
+    # DER aggregator revenue by year (under VIU)
+    revenue::ParamArray
```

Comment:
Couldn't the DERAggregator also register their total revenue under the WM structure? It seems like we'd want that as an output?